<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>receipt</class>
 <widget class="QDialog" name="receipt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transaction Recipt</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    background-color:	#3b3b3b;

	color: #fff;
	selection-background-color: #fff;
	selection-color: #000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #fff;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #00aae4;
	color: #fff;
	min-width: 80px;
	border-radius: 4px;
	padding: 5px;

}



QPushButton::hover
{
	background-color: #0486b2;
	border: 1px solid #0486b2;

}



</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>251</width>
     <height>99</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="aNumber">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Previous Balance:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="preBalance">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Account Number:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="newBalance">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Current Balance:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>40</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - --</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>91</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS PGothic</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Transaction Receipt</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
